tests/test_driver.py

```diff
@@ -334,12 +318,6 @@ def test_config_file(self):
         self.assertEqual(arguments.transactions, True)
         config_file.close()
 
-    @patch.object(mintapi.signIn, "get_web_driver")
-    def test_build_bundledServiceController_url(self, mock_driver):
```

Comment:
What does it mean for a test method to be unused? Isn't this test running?

Reply:
The bundled service method is no longer needed and so this test is no longer needed as well.